spatial-partition/src/main/java/com/iluwatar/spatialpartition/QuadTree.java

```diff
@@ -34,6 +35,8 @@
  */
 
 public class QuadTree {
+  public static boolean[] coveredBranches = new boolean[8];
```

Comment:
Initialise this the test in a `@BeforeAll` to keep the state between the different tests.